crates/services/upgradable-executor/src/error.rs

```diff
@@ -0,0 +1,13 @@
+/// Error type for the upgradable (wasm) executor.
+#[cfg(feature = "wasm-executor")]
+#[allow(missing_docs)]
+#[derive(Debug, Clone, PartialEq, derive_more::Display, derive_more::From)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
Do we really need `serde` here?

Reply:
No but I though it wouldn't hurt either. Might need that in the future. Probably not, though. Removed in 6d9309a16.